Atomic/trust.py

```diff
@@ -143,6 +145,8 @@ def delete(self):
         with open(self.policy_filename, 'r+') as policy_file:
             policy = json.load(policy_file)
             try:
+                if self.args.remove:
+                    sigstore = policy["transports"][sstype][self.args.registry]["sigstore"]
```

Comment:
Should be calling `util.get_registry_configs()` here, then removing the yaml section from the returned filename.